sdk/containerregistry/azure-containerregistry/tests/testcase.py

```diff
@@ -74,6 +105,11 @@ def process_request(self, request):
         if request.body:
             request.body = self._scrub_body(request.body)
 
+        if "seankane.azurecr.io" in request.uri:
+            request.uri = request.uri.replace("seankane.azurecr.io", "fake_url.azurecr.io")
+        if "seankane.azurecr.io" in request.url:
+            request.url = request.url.replace("seankane.azurecr.io", "fake_url.azurecr.io")
```

Comment:
In form, we've been able to handle this in one of the custom response scrubbers: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/tests/testcase.py#L68-L72

I'm not sure if this would work in the request, though. Is this service set up such that "seankane.azurecr.io" is always used as the test resource or if I ran it would be "kristapratico.azurecr.io" and not get scrubbed?

Reply:
@kristapratico I need to add a similar scrubber as the one you have for formrecognizer, thank you for pointing me towards that.

Reply:
I opened a tracking issue for this FYI, I am going to mock what formrecognizer is doing. I will also look into adding this type of recording processor to the default for `AzureTestCase`

Reply:
sounds good, thanks!